src/cljs/villagebookUI/helpers.cljs

```diff
@@ -41,3 +41,10 @@
        (on-success))
      (fn [res]
        (show-error (:response res))))))
+
+(defn show-alert-bottom
+  [status message & [time]]
+  (let [time (if (number? time) (* time 1000) 4000)]
```

Comment:
`number?` seems like the wrong check here. are you checking for presence?

Reply:
Checking if time is a number and is not nil.